events/yaml-sync.js

```diff
@@ -20,6 +20,19 @@ function getRegion (region) {
   return reg
 }
 
+function isSoT (region, city, name) {
+  let meetups = region.meetups
```

Comment:
I think we can remove this line if the first argument of the function is changed to `meetups`.